scripts/gen-netatt.sh

```diff
@@ -277,9 +277,9 @@ spec:
       "master": "${INTERFACE}.$((${VLAN_START}+${VLAN_STEP}*4))",
       "ipam": {
         "type": "whereabouts",
-        "range": "172.30.0.0/24",
-        "range_start": "172.30.0.30",
-        "range_end": "172.30.0.70"
+        "range": "172.24.0.0/24",
```

Comment:
where it is on it's own vlan it might not be a problem but this is the same range as octavia

Reply:
Ah damn, let me change it. I'll take a better look at what we're already using and make sure it doesn't conflict with anything else.

Reply:
`172.28` seems to be free.